src/components/menuBar/menu-bar.spec.js

```diff
@@ -26,11 +26,27 @@ describe('material.components.menuBar', function() {
       });
 
       describe('ARIA', function() {
+        
         it('sets role="menubar" on the menubar', function() {
           var menuBar = setup();
           var ariaRole = menuBar[0].getAttribute('role');
           expect(ariaRole).toBe('menubar');
         });
+        
+        it('should set the role on the menu trigger correctly', inject(function($compile, $rootScope) {
```

Comment:
I can't find it anymore, but I thought that the version I was saying was in the official style guide that we follow.

It's not a big deal, just more of a personal preference 😄